### DISCUSSION TITLE
Syntax differences between MySQL and DBeaver

### DISCUSSION THREAD

**Discussion by unceasingmalice** [dbeaver/dbeaver#23316] [Q&A] [answered]
### New to DBeaver, new to SQL

Hi. I have a question about DBeaver syntax. 

SELECT * FROM table
WHERE amount > 10000;

Why does it work without problems in all SQL Syntax Checker/Validator/Analyzer and in software like MySQL Workbench, but DBeaver shows me a syntax error?

**Comment by LonwoLonwo** [comment]
Hello @unceasingmalice 

Please provide a screen of your SQL Editor and error from the error log viewer.

**Comment by unceasingmalice** [comment, marked as answer]
Hey @LonwoLonwo! Thanks for your reply. 
It was an error related to an **empty string** between `SELECT * FROM table` and `WHERE amount > 10000;`.

atm I've configured the app to not recognise empty strings as a syntax error.

fyi: in my opinion that setting should be turned off by default, cuz for example as a newbie I spent time googling to make sure that **in all similar software the empty strings doesn't trigger as syntax error** :/

**Reply by mkgrgis** [reply]
DBeaver threat empty strings (>2 `\n` characters) between query parts as possible begin of a new query. Not only `;` character treated in DBeaver as end of a query.
